src/Autosuggest.js

```diff
@@ -235,15 +252,19 @@ class Autosuggest extends Component {
           case 'Enter': {
             const focusedSuggestion = this.getFocusedSuggestion();
 
-            if (focusedSuggestion !== null) {
+            if (focusedSuggestion) {
```

Comment:
Could you describe how to get to the state when `focusedSuggestion === undefined`?
Did the fact that we didn't check for `undefined` caused any issues?
Do we have a missing test case?